File: 2.0/java/campaigns/postTTSCampaignSound.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.TextToSpeech;
import com.callfire.api.client.api.campaigns.model.Voice;
import com.callfire.api.client.api.common.model.ResourceId;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        TextToSpeech tts = new TextToSpeech();
        tts.setVoice(Voice.MALE1);
        tts.setMessage("This is a TTS sound");
        ResourceId resourceId = callfireClient.campaignSoundsApi().createFromTts(tts);
    }
}

File: 2.0/csharp/campaigns/postTTSCampaignSound.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var tts = new TextToSpeech
        {
            Message = "This is a TTS sound",
            Voice = Voice.MALE1
        };
        ResourceId resourceId = client.CampaignSoundsApi.CreateFromTts(tts);
    }
}
